tests/playwright/src/spec/minikube-extension.spec.ts

```diff
@@ -88,6 +106,34 @@ test.describe.serial('Podman Desktop Minikube Extension Tests', () => {
         await ensureCliInstalled(page, 'Minikube');
     });
 
+    test.describe('Minikube cluster e2e test', () => {
+        test('Create a Minikube cluster', async ({ page}) => {
+            test.setTimeout(CLUSTER_CREATION_TIMEOUT);
+            await createMinikubeCluster(page, CLUSTER_NAME, true, CLUSTER_CREATION_TIMEOUT);
+          });
+      
+          test('Check resources added with the Minikube cluster', async ({ page }) => {
+            await checkClusterResources(page, MINIKUBE_CONTAINER);
+          });
+      
+          test('Minikube cluster operations - STOP', async ({ page }) => {
+            await resourceConnectionAction(page, minikubeResourceCard, ResourceElementActions.Stop, ResourceElementState.Off);
+          });
+      
+          test('Minikube cluster operations - START', async ({ page }) => {
+            await resourceConnectionAction(
+              page,
+              minikubeResourceCard,
+              ResourceElementActions.Start,
+              ResourceElementState.Running,
+            );
+          });
+      
```

Comment:
You have added restart test as well, and it does not work. So I would add a skip into it and create a follow up to resolve / have issue tracking problem with restart.